Makefile

```diff
@@ -95,6 +98,19 @@ release-manifests:
 	kubectl kustomize manifests/overlays/release-ha > _output/high-availability.yaml
 	kubectl kustomize manifests/overlays/release-ha-1.21+ > _output/high-availability-1.21+.yaml
 
+.PHONY: release-binaries
+release-binaries:
+	@mkdir -p _output
+	@for arch in $(BINARIES_ARCHITECTURES); do \
+		for platform in $(BINARIES_PLATFORMS); do \
```

Comment:
I think it would be cleaner to have two separate rules instead of having code here. In a similar fashion to what we do when building container for all the arch: https://github.com/kubernetes-sigs/metrics-server/blob/master/Makefile#L46-L58